test/unit/gateways/airwallex_test.rb

```diff
@@ -179,15 +179,24 @@ def test_purchase_passes_request_id
     stub_comms do
       @gateway.purchase(@amount, @credit_card, @options.merge(request_id: request_id))
     end.check_request do |_endpoint, data, _headers|
-      assert_match(/request_/, data)
+      assert_match(request_id, data)
```

Comment:
This change is unrelated to the `descriptor` field. Because the request contains a field named `request_id`, this assertion was not really testing the presence of the `request_{timestamp}` value.

Reply:
Nice, thanks for the description 😉